db/migrate/20200210115945_create_users.rb

```diff
@@ -0,0 +1,16 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    enable_extension("citext")
+
+    create_table :users do |t|
+      t.citext :email
```

Comment:
```suggestion
      t.citext :email, null: false
```

Should we add null db constraint?